components/TopNav.vue

```diff
@@ -1,16 +1,32 @@
 <template>
-  <div class="p-4 border-b flex justify-between">
-    <div class="font-semibold text-xl text-gray-800">Find a Doc</div>
-    <div>
-      <a href="https://ko-fi.com/Z8Z14YNUR" target="_blank"
-        ><img
-          height="36"
-          style="border: 0px; height: 36px"
-          src="https://cdn.ko-fi.com/cdn/kofi1.png?v=3"
-          border="0"
-          alt="Buy Me a Coffee at ko-fi.com"
-      /></a>
+    <div class="p-4 border-b flex justify-between">
+        <div class="font-semibold text-xl text-gray-800">Find a Doc</div>
+        <div>
+            <NuxtLink to="/">Home</NuxtLink>
+            <NuxtLink to="/aboutPage">About</NuxtLink>
+            <NuxtLink to="/newsPage">News</NuxtLink>
+            <a href="https://ko-fi.com/Z8Z14YNUR" target="_blank">
+                <img height="36" style="border: 0px; height: 36px" src="https://cdn.ko-fi.com/cdn/kofi1.png?v=3"
+                    alt="Buy Me a Coffee at ko-fi.com" />
+            </a>
+        </div>
     </div>
-  </div>
 </template>
 
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+    props: {
+        msg: {
+            type: String,
+            required: false
+        }
+    },
+    data() {
+        return {
+            name: '',
+        }
+    }
+})
+</script>
```

Comment:
I don't want to check these `<script>` sections in for a few reasons.

1. They aren't currently being used and are just adding unnecessary lines to the code.
2. They might encourage people to start importing and exporting components the Vue way instead of the Nuxt way, which defeats one of the biggest reasons for using Nuxt.

For now, please remove these. 🙏🏾

Reply:
these files are literally for my personal use at the moment and I'm going to be filling out a lot of logic in these script sections. For now these are placeholders that I'm actively filling out locally until this can get merged in. 

This PR is just templating, not final files.